### PR TITLE
Add NWB export function

### PR DESCRIPTION
**Pull request by CBroz1**
Here I add an NWB export function. Before merge...
1. [These changes](https://github.com/catalystneuro/DLC2NWB/pull/1) will need to be PR'd to the main repo. Currently under review with some of the core NWB team before bringing to the attention of DLC folks.
2. We should evaluate whether or not to incorporate training data here or in a future PR. [This PR](https://github.com/rly/ndx-pose/pull/9) reflects the changes in the NWB standard that would permit storing training data.

### CODE REVIEW

**Comment by CBroz1** [comment]
Closing to test revised CI/CD

**Comment by CBroz1** [comment]
Hi @tdincer - because my [DLC2NWB PR](https://github.com/DeepLabCut/DLC2NWB/pull/16) was merged, this PR is now ready for review. It allows for export under the current nwb pose estimation extension and does not yet incorporate [proposed changes to include training info](https://github.com/rly/ndx-pose/pull/9)

File: element_deeplabcut/export/nwb.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+    Calls DLC2NWB to export NWB file using current h5 on disk. If use_element_session,
+    calls NWB export function from Elements for lab, animal and session, passing
+    session_kwargs. Saves output based on naming convention in DLC2NWB. If output path
+    already exists, returns output path without making changes to the file.
+    NOTE: does not support multianimal exports
+
+    Parameters
+    ----------
+    Keys - one or more keys from model.PoseEstimation
```
Comment (tdincer) [inline element_deeplabcut/export/nwb.py:35]:
```suggestion
    keys:
```

Comment (tdincer) [reply element_deeplabcut/export/nwb.py:35]: The keyword is defined in small letters.

File: element_deeplabcut/export/nwb.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    calls NWB export function from Elements for lab, animal and session, passing
+    session_kwargs. Saves output based on naming convention in DLC2NWB. If output path
+    already exists, returns output path without making changes to the file.
+    NOTE: does not support multianimal exports
+
+    Parameters
+    ----------
+    keys:
+    use_element_session - Optional. If True, call NWB export from Element Session
+    session_kwargs - Optional. Additional keyword arguements for Element Session export
```
Comment (CBroz1) [inline element_deeplabcut/export/nwb.py:37]:
```suggestion
    keys: One or more keys from model.PoseEstimation
    use_element_session: Optional. If True, call NWB export from Element Session
    session_kwargs: Optional. Additional keyword arguments for Element Session export
```

**Pull request merged**